src/pspm_find_sounds.m

```diff
@@ -20,19 +20,17 @@
 % │                 each channel.  In this case, set the option 
 % │                 'channel_action' to 'add',  to store each
 % │                 resulting channel separately.
-% ├.channel_action: ['none'/'add'/'replace'] if not set to 'none'
+% ├.channel_action: ['add'/'replace'] 
```

Comment:
We may want to discuss about whether we would like to keep the optional value `none` for `channel_action` ?